salt/modules/win_certutil.py

```diff
@@ -122,7 +122,7 @@ def add_store(source, store, retcode=False, saltenv="base"):
         return __salt__["cmd.run"](cmd)
 
 
-def del_store(source, store, retcode=False, saltenv="base"):
+def del_store(source, store, saltenv="base", retcode=False):
```

Comment:
Instead of changing the position of the kwargs, can we just call the kwarg from the call in the state. I'll include example in the state module below.